Move all bridge driver logging to dynamic logging (pr_debug)

diff --git a/net/bridge/br_stp.c b/net/bridge/br_stp.c
index 4a36459..9ad8595 100644
--- a/net/bridge/br_stp.c
+++ b/net/bridge/br_stp.c
@@ -31,7 +31,7 @@ static const char *const br_port_state_names[] = {
 
 void br_log_state(const struct net_bridge_port *p)
 {
-	br_info(p->br, "port %u(%s) entered %s state\n",
+	br_debug(p->br, "port %u(%s) entered %s state\n",
 		(unsigned int) p->port_no, p->dev->name,
 		br_port_state_names[p->state]);
 }
